documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
```

Comment:
It really isn't. See https://en.m.wikipedia.org/wiki/Instrument_landing_system.

An aircraft approaching a runway is guided by the ILS receivers in the aircraft by performing modulation depth comparisons. Many aircraft can route signals into the autopilot to fly the approach automatically. An ILS consists of two independent sub-systems. **The localiser provides lateral guidance; the glide slope provides vertical guidance.**